src/Header/Account/nice-accounts.js

```diff
@@ -1,7 +1,8 @@
 // Checks if you are logged in via NICE Accounts
 // Returns a promise that resolves with the data from NICE Accounts.
 // Returns a promise that rejects if the data could not be loaded.
-export const checkIsLoggedIn = function(environment) {
+
+export const checkIsLoggedIn = environment => {
```

Comment:
I think there's an issue with transpilation of arrow functions working probably in IE8. Although we no longer officially support IE8, the global nav contains a message telling people that, so we don't want it to break if we can avoid it. So please can you revert this to a normal function?